### DISCUSSION TITLE
Advanced toggle within system not visible.

### DISCUSSION THREAD

**Discussion by 21st-centuryman** [utmapp/UTM#5071] [UTM Help (macOS)] [answered]
Hello I am unable to find the advanced toggle for any of my VM's (see attatched picture). I am wondering why that is as I have found little to no information on what would make this button appear for my VM's. As in does it require special privileged. I have downloaded UTM through homebrew if that helps the issue.

<img width="914" alt="image" src="https://user-images.githubusercontent.com/60197499/215330270-ce049173-3469-4339-95bd-87e9500b08d3.png">

**Comment by osy** [comment, marked as answer]
You are referencing an older version of UTM. Please refer to https://docs.getutm.app/settings-qemu/settings-qemu/ for the settings on the latest version.
